datahub-web-react/src/app/entity/user/UserInfoSideBar.tsx

```diff
@@ -120,6 +126,7 @@ export default function UserInfoSideBar({ sideBarData, refetch }: Props) {
                 <SideBarSubSection className={isProfileOwner ? '' : 'fullView'}>
                     <CustomAvatar size={160} photoUrl={photoUrl} name={avatarName} style={AVATAR_STYLE} />
                     <Name>{name || <EmptyValue />}</Name>
+                    <UserDetails>{urn?.replace('urn:li:corpuser:', '') || <EmptyValue />}</UserDetails>
```

Comment:
Can you please revert this change? Why is this happening? This is just to get the user id, i guess?

Reply:
Also, why is this in a div called <UserDetails>?

Maybe we should call it <UserId> since that aligns with name. 

Final comment. We already return a field called `username` in the GraphQL response for users. Let's instead use that as opposed to snipping the URN like this. Generally we don't like "inspecting" an URN from the outside - it leads to bad coding practices and fragility. 

For example, you can use this field: 

![Screenshot 2024-12-09 at 1 15 44 PM](https://github.com/user-attachments/assets/c7327e41-f58f-421b-afed-4a807b098aa9)

Reply:
`username`